CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+- Collect information about the bug:
+- Stack trace (Traceback)
+- OS, Platform and Version (Windows, Linux, macOS, x86, ARM)
+- Version of the interpreter, compiler, SDK, runtime environment, package manager, depending on what seems relevant.
+- Possibly your input and the output
+
+#### How Do I Submit a Good Bug Report?
+
+> Please do not report security related issues, vulnerabilities or bugs including sensitive information to the issue tracker, or elsewhere in public. Instead sensitive bugs should be sent by email to perseus@ib1.org
+
+We use GitHub issues to track bugs and errors. If you run into an issue with the project:
+
+- Open an [Issue](https://github.com/icebreakerone/perseus-demo-energy/issues/new). (Since we can't be sure at this point whether it is a bug or not, we ask you not to talk about a bug yet and not to label the issue.)
+- Explain the behavior you would expect and the actual behavior.
+- Please provide as much context as possible and describe the _reproduction steps_ that someone else can follow to recreate the issue on their own. This usually includes your code. For good bug reports you should isolate the problem and create a reduced test case.
+- Provide the information you collected in the previous section.
+
+Once it's filed, we will label the issue accordingly and work on it as soon as possible.
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion for Perseus Demo Energy, **including completely new features and minor improvements to existing functionality**.
+
+#### Before Submitting an Enhancement
+
+- Make sure that you are using the latest version.
+- Read the [documentation](https://github.com/icebreakerone/perseus-demo-energy/blob/main/README.md) carefully and find out if the functionality is already covered, maybe by an individual configuration.
+- Perform a [search](https://github.com/icebreakerone/perseus-demo-energy/issues) to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
+- Find out whether your idea fits with the scope and aims of the project. If you are not sure, feel free to ask in the [slack channel](#asking-questions).
+
+#### How Do I Submit a Good Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://github.com/icebreakerone/perseus-demo-energy/issues).
+
+- Use a **clear and descriptive title** for the issue to identify the suggestion.
+- Provide a **step-by-step description of the suggested enhancement** in as many details as possible.
+- **Describe the current behavior** and **explain which behavior you expected to see instead** and why
+- Provide links to code, if applicable
+
+### Improving The Documentation
+
+Feedback on our documentation is always welcome. If you see something that is wrong, missing, or could be explained better, you can help us by creating an issue, or by creating a pull request with the suggested changes.
```

Comment:
Not sure if we should put something about other channels for comments on documentation? Thinking that anyone reading this is probably a github user, so will be able to deal with raising issues / pull requests